tests/unit/test_python_evaluation.py

```diff
@@ -3,6 +3,7 @@
 
 import pandas as pd
 import pytest
+from reco_utils.common.constants import *
```

Comment:
Would you please add the names instead of *? It's easier to debug and understand the code that way